src/roswire/ros2/msg.py

```diff
@@ -35,6 +38,16 @@ def from_string(cls, package: str, line: str) -> Optional["ROS2Field"]:
             return field
         return None
 
+    @classmethod
+    def _resolve_type(cls, package: str, typ: str) -> str:
+
+        # The string bounds (string<=123) will be removed to
+        # help resolution of the type
+        r_type = cls.REXP_TYPE.match(typ)
```

Comment:
prefer `match` to indicate that the result of the method call is an `re.Match` rather than a regular expression